File: packages/style-analyzer/fixtures/structure-sample.py
# Builtin imports
import os
import sys
from pathlib import Path

# External imports
import requests
from pydantic import BaseModel

# Relative imports
from .utils import helper
from ..constants import MAX_SIZE


# Named functions (no default export concept in Python)
def process_data(input_str: str) -> str:
    return input_str.strip()


VERSION = "1.0.0"


class DataProcessor:
    def process(self, input_str: str) -> str:
        return input_str

File: packages/style-analyzer/fixtures/structure-sample.ts
// @ts-nocheck
// Builtin imports
import * as path from "node:path";
import * as fs from "node:fs";

// External imports
import { z } from "zod";
import chalk from "chalk";

// Internal imports (aliases)
import { UserService } from "@app/services/user";
import type { Config } from "@app/config";

// Relative imports
import { helper } from "./utils";
import { CONSTANTS } from "../constants";

// Named exports (inline)
export function processData(input: string): string {
  return input.trim();
}

export const VERSION = "1.0.0";

export interface DataResult {
  value: string;
  status: number;
}

// Default export
export default class DataProcessor {
  process(input: string) {
    return input;
  }
}
